lib/rts2tel/teld.cpp

```diff
@@ -1494,7 +1494,11 @@ void Telescope::checkMoves ()
 			}
 			else
 			{
-				maskState (TEL_MASK_CORRECTING | TEL_MASK_MOVING | BOP_EXPOSURE, TEL_NOT_CORRECTING | TEL_OBSERVING, "move finished without error");
+				if ((woffsRaDec->wasChanged () || wcorrRaDec->wasChanged ()))
```

Comment:
Is this really useful? This might solve a situation, when wait-offsets or wait-corrections changed during the (just finished) movement. Anything defined before should already be included in this movement... After new movement started, no (astrometry) feedback is taken into account, so the only thing which can change is the wait-offsets (which is not very probable)...?

Reply:
Yes, I believe I made this exactly to solve such situations. Do not remember the details, maybe that was useful for solar system targets (which we observe a lot), or to applying small adjustments between exposures. It was definitely to prevent something real that caused trailed images in our setup (as the exposure was triggered on releasing the block, but the mount immediately set the block again and started to move)

Reply:
Hmmm, it's strange. This change is probably harmless, but still, IMO it's not systemic and might be confusing...
Also there were some errors in the teld, maybe this is not necessary to have it there now. Could you put there some log message and try to find it in your logfiles after several nights of observations?

Reply:
Probably the reason here is what tracking MPEC objects actually does not increase moveNum - so the corrections from previous pointings are still valid, and may arrive during the repointing. I added some logging to the code, let's see whether this path will be called...